pkgs/applications/networking/instant-messengers/teamspeak/distribution-permit.txt

```diff
@@ -0,0 +1,32 @@
+License issues:
```

Comment:
Suggestions for better file names are welcome.

Reply:
Sounds good to me.